roles/hammer_credentials/templates/zzz_foreman_forklift_credentials.yml.j2

```diff
@@ -0,0 +1,8 @@
+:foreman:
+  # Server connection setup
+  :host: {{ hammer_creadentials_host }}
+  :username: {{ hammer_creadentials_username }}
+  :password: {{ hammer_creadentials_password }}
+
+:ssl:
+  :verify_ssl: false
```

Comment:
I'd prefer to enable SSL verification by default. Production should also use it and this could help us detect problems early on. If needed you can point to a CA file, which is what ```foreman::cli``` in the ```puppet-foreman``` module does.